lib/prepare-build-files.js

```diff
@@ -0,0 +1,70 @@
+const path = require('path');
+const fse = require('fs-extra');
+
+function copyReadme() {
+  return fse
+    .copyFile(
+      path.resolve(__dirname, '..', 'README.md'),
+      path.resolve(__dirname, 'build', 'README.md')
+    )
+    .then(() => console.log('> Copied README.md'));
+}
+
+function createAdapterPackageFile(name) {
+  const packageJson = {
+    name: `@material-ui/pickers-adapter-${name}`,
```

Comment:
What do you think will be the best name for adapter package.json? 
BTW `@material-ui/pickers/adapter/${name}` is not a valid name for package.json

Reply:
Do we even need to provide a name? For instance: https://unpkg.com/browse/@material-ui/core@4.9.9/Badge/package.json.

Reply:
We had https://github.com/mui-org/material-ui/issues/15715, but it seemed fine.